starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,30 @@
+body {
+  font-family: "Public Sans", sans-serif;
+  font-size: 1.1em;
+}
+
+.header-container {
+  background-image: url("/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg");
+  background-size: cover;
+  height: 60vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.header-container h1 {
+  color: white;
+  text-align: center;
+  font-size: 4em;
+}
+
+img {
```

Comment:
You are giving the same styling to all img. Best is to give a class name to the image you want to change the styling for and then reference it in css using that specific class.